### PR TITLE
tiup: add kvcdc server topylogy example

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

topology config supports TiCDC server and TiKV-CDC, they are different.
https://asktug.com/t/topic/1034131
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/19230
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-cluster-topology-reference.md [hunk 1]
```diff
@@ -365,6 +366,39 @@ tiproxy_servers:
   - host: 10.0.1.22
 ```
 
+### `kvcdc_servers`
+
+`kvcdc_servers` 约定了将 [TiKV-CDC](https://tikv.org/docs/latest/concepts/explore-tikv-features/cdc/cdc-cn/) 服务部署到哪些机器上，同时可以指定每台机器上的服务配置，`kvcdc_servers` 是一个数组，每个数组元素包含以下字段：
+
+- `addr`：TiKV-CDC 的监听地址，用于提供 HTTP API 和 Prometheus 查询，默认为 `127.0.0.1:8600`。
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:373]: 这句的英文是 “- `addr`: The listening address of TiKV-CDC, the HTTP API address, and the Prometheus address of the TiKV-CDC service. The default value is `127.0.0.1:8600`.“， ”用于提供 HTTP API 和 Prometheus 查询"看起来和英文不太对的上

Comment (wk989898) [reply tiup/tiup-cluster-topology-reference.md:373]: 修改了英文

**Review by wk989898 — APPROVED** [review]

File: tiup/tiup-cluster-topology-reference.md [hunk 2]
```diff
@@ -365,6 +366,39 @@ tiproxy_servers:
...
+- `advertise-addr`：TiKV-CDC 供客户端访问的外部开放地址。如果未设置，默认与 `addr` 值相同。
+- `pd`：TiKV-CDC 监听的 PD 节点地址，多个地址之间用英文逗号（`,`）分隔。
+- `config`：可选项，指定 TiKV-CDC 使用的配置文件路径。
+- `data-dir`：可选项，指定 TiKV-CDC 存储运行时数据的目录，主要用于外部排序。建议确保该目录所在磁盘的可用空间大于等于 500 GiB。
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:377]:
```suggestion
- `data-dir`：可选项，指定 TiKV-CDC 存储临时文件的目录，主要用于外部排序。建议确保该目录所在磁盘的可用空间大于等于 500 GiB。
```

File: tiup/tiup-cluster-topology-reference.md [hunk 3]
```diff
@@ -365,6 +366,39 @@ tiproxy_servers:
...
 
+### `kvcdc_servers`
+
+`kvcdc_servers` 约定了将 [TiKV-CDC](https://tikv.org/docs/latest/concepts/explore-tikv-features/cdc/cdc-cn/) 服务部署到哪些机器上，同时可以指定每台机器上的服务配置，`kvcdc_servers` 是一个数组，每个数组元素包含以下字段：
+
+- `addr`：TiKV-CDC 的监听地址，用于提供 HTTP API 和 Prometheus 查询，默认为 `127.0.0.1:8600`。
+- `advertise-addr`：TiKV-CDC 供客户端访问的外部开放地址。如果未设置，默认与 `addr` 值相同。
+- `pd`：TiKV-CDC 监听的 PD 节点地址，多个地址之间用英文逗号（`,`）分隔。
+- `config`：可选项，指定 TiKV-CDC 使用的配置文件路径。
+- `data-dir`：可选项，指定 TiKV-CDC 存储临时文件的目录，主要用于外部排序。建议确保该目录所在磁盘的可用空间大于等于 500 GiB。
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:377]:
```suggestion
- `data-dir`：可选项，指定 TiKV-CDC 存储临时文件的目录，主要用于排序。建议确保该目录所在磁盘的可用空间大于等于 500 GiB。
```

File: tiup/tiup-cluster-topology-reference.md [hunk 4]
```diff
@@ -365,6 +366,39 @@ tiproxy_servers:
   - host: 10.0.1.22
 ```
 
+### `kvcdc_servers`
+
+`kvcdc_servers` 约定了将 [TiKV-CDC](https://tikv.org/docs/latest/concepts/explore-tikv-features/cdc/cdc-cn/) 服务部署到哪些机器上，同时可以指定每台机器上的服务配置，`kvcdc_servers` 是一个数组，每个数组元素包含以下字段：
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:371]:
```suggestion
`kvcdc_servers` 约定了将 [TiKV-CDC](https://tikv.org/docs/latest/concepts/explore-tikv-features/cdc/cdc-cn/) 服务部署到哪些机器上，同时可以指定每台机器上的服务配置。`kvcdc_servers` 是一个数组，每个数组元素包含以下字段：
```

File: tiup/tiup-cluster-topology-reference.md [hunk 5]
```diff
@@ -365,6 +366,39 @@ tiproxy_servers:
...
+- `data-dir`：可选项，指定 TiKV-CDC 存储临时文件的目录，主要用于排序。建议确保该目录所在磁盘的可用空间大于等于 500 GiB。
+- `gc-ttl`：可选项，TiKV-CDC 在 PD 设置服务级别 GC safepoint 的 TTL (Time to Live) 时长。同时也是 TiKV-CDC 同步任务暂停的最大时长。单位为秒，默认值为 86400，即 24 小时。注意：TiKV-CDC 同步任务的暂停会影响集群 GC safepoint 的推进。`gc-ttl` 越大，同步任务可以暂停的时间越长，但同时会保留更多的过期数据，并占用更多的存储空间。反之亦然。
+- `log-file`：可选项，TiKV-CDC 进程运行时日志的输出路径，未设置时默认为标准输出 (stdout)。
+- `log-level`：可选项，TiKV-CDC 进程运行时的日志路径，默认为 `"info"`。
+- `ca`：可选项，指定用于 TLS 连接的 CA 证书文件路径。仅支持 PEM 格式。
+- `cert`：可选项，指定用于 TLS 连接的证书文件路径。仅支持 PEM 格式。
+- `key`：可选项，指定用于 TLS 连接的私钥文件路径。仅支持 PEM 格式。
+- `cert-allowed-cn`：可选项，指定允许的调用者身份（即证书 Common Name，CN）。多个 CN 之间用英文逗号（`,`）分隔。
+
+以上所有字段中，部分字段部署完成之后不能再修改。如下所示：
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:386]: 这句话好像不对，下面的字段中只有 `data_dir` 出现在了“以上字段中”。

File: tiup/tiup-cluster-topology-reference.md [hunk 6]
```diff
@@ -365,6 +366,39 @@ tiproxy_servers:
...
+### `kvcdc_servers`
+
+`kvcdc_servers` 约定了将 [TiKV-CDC](https://tikv.org/docs/latest/concepts/explore-tikv-features/cdc/cdc-cn/) 服务部署到哪些机器上，同时可以指定每台机器上的服务配置，`kvcdc_servers` 是一个数组，每个数组元素包含以下字段：
+
+- `addr`：TiKV-CDC 的监听地址，用于提供 HTTP API 和 Prometheus 查询，默认为 `127.0.0.1:8600`。
+- `advertise-addr`：TiKV-CDC 供客户端访问的外部开放地址。如果未设置，默认与 `addr` 值相同。
+- `pd`：TiKV-CDC 监听的 PD 节点地址，多个地址之间用英文逗号（`,`）分隔。
+- `config`：可选项，指定 TiKV-CDC 使用的配置文件路径。
+- `data-dir`：可选项，指定 TiKV-CDC 存储临时文件的目录，主要用于排序。建议确保该目录所在磁盘的可用空间大于等于 500 GiB。
+- `gc-ttl`：可选项，TiKV-CDC 在 PD 设置服务级别 GC safepoint 的 TTL (Time to Live) 时长。同时也是 TiKV-CDC 同步任务暂停的最大时长。单位为秒，默认值为 86400，即 24 小时。注意：TiKV-CDC 同步任务的暂停会影响集群 GC safepoint 的推进。`gc-ttl` 越大，同步任务可以暂停的时间越长，但同时会保留更多的过期数据，并占用更多的存储空间。反之亦然。
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:378]:
```suggestion
- `gc-ttl`：可选项，TiKV-CDC 在 PD 设置服务级别 GC safepoint 的 TTL (Time to Live) 时长。同时也是 TiKV-CDC 同步任务暂停的最大时长。单位为秒，默认值为 `86400`，即 24 小时。注意：TiKV-CDC 同步任务的暂停会影响集群 GC safepoint 的推进。`gc-ttl` 越大，同步任务可以暂停的时间越长，但同时会保留更多的过期数据，并占用更多的存储空间。反之亦然。
```

File: tiup/tiup-cluster-topology-reference.md (hunk 5 above)
Comment (qiancai) [reply tiup/tiup-cluster-topology-reference.md:386]: @wk989898 PTAL

Comment (qiancai) [reply tiup/tiup-cluster-topology-reference.md:386]: updated in [`9382ec1` (#18927)](https://github.com/pingcap/docs-cn/pull/18927/commits/9382ec1f85fb84f929bd4dcd5a0cbea0642169da)

Comment (wk989898) [reply tiup/tiup-cluster-topology-reference.md:386]: LGTM

File: tiup/tiup-cluster-topology-reference.md [hunk 7]
```diff
@@ -365,6 +366,41 @@ tiproxy_servers:
...
+- `arch`：`host` 字段所指定的机器的架构，若不指定该字段，则默认为 `global` 中的 `arch`。
+- `resource_control`：针对该服务的资源控制，如果配置了该字段，会将该字段和 `global` 中的 `resource_control` 内容合并（若字段重叠，以本字段内容为准），然后生成 systemd 配置文件并下发到 `host` 指定机器。`resource_control` 的配置规则同 `global` 中的 `resource_control`。
+
+以上所有字段中，部分字段部署完成之后不能再修改。如下所示：
+
+- `host`
+- `port`
+- `deploy_dir`
+- `data_dir`
+- `log_dir`
```
Comment (hfxsd) [inline tiup/tiup-cluster-topology-reference.md:392]: `log_dir` 没有出现在字段解释里，要删掉吗

File: tiup/tiup-cluster-topology-reference.md [hunk 8]
```diff
@@ -365,6 +366,41 @@ tiproxy_servers:
...
 
+### `kvcdc_servers`
+
+`kvcdc_servers` 约定了将 [TiKV-CDC](https://tikv.org/docs/latest/concepts/explore-tikv-features/cdc/cdc-cn/) 服务部署到哪些机器上，同时可以指定每台机器上的服务配置。`kvcdc_servers` 是一个数组，每个数组元素包含以下字段：
+
+- `host`：指定 TiKV-CDC 部署到哪台机器，字段值填 IP 地址，不可省略。
+- `ssh_port`：指定连接目标机器进行操作时使用的 SSH 端口，若不指定，则使用 `global` 区块中的 `ssh_port`。
+- `port`：TiKV-CDC 服务的监听端口，默认 8600。
+- `deploy_dir`：指定部署目录，若不指定，或指定为相对目录，则按照 `global` 中配置的 `deploy_dir` 生成。
+- `data-dir`：可选项，指定 TiKV-CDC 存储临时文件的目录，主要用于排序。建议确保该目录所在磁盘的可用空间大于等于 500 GiB。
```
Comment (qiancai) [inline tiup/tiup-cluster-topology-reference.md:377]:
```suggestion
- `data-dir`：可选项，指定 TiKV-CDC 存储临时文件的目录，主要用于排序。建议确保该目录所在磁盘的可用空间大于等于 500 GiB。
- `log_dir`：指定日志目录，若不指定，或指定为相对目录，则按照 `global` 中配置的 `log_dir` 生成。
```

File: tiup/tiup-cluster-topology-reference.md (hunk 7 above)
Comment (qiancai) [reply tiup/tiup-cluster-topology-reference.md:393]: added in [`1517850` (#18927)](https://github.com/pingcap/docs-cn/pull/18927/commits/15178502d09550569e589b934d3f7cfff228b605)

**Review by hfxsd — APPROVED** [review]

**Pull request merged**